docs/data-sources/mirrormaker_replication_flow.md

```diff
@@ -33,6 +33,7 @@ data "aiven_mirrormaker_replication_flow" "f1" {
 
 ### Read-Only
 
+- `config_properties_exclude` (List of String) List of topic configuration properties and/or regular expressions to not replicate. The properties that are not replicated by default are: `follower.replication.throttled.replicas`, `leader.replication.throttled.replicas`, `message.timestamp.difference.max.ms`, `message.timestamp.type`, `unclean.leader.election.enable`, and `min.insync.replicas`. Setting this overrides the defaults. For example, to exclude only 'min.insync.replicas' and 'unclean.leader.election.enable'  set this to: ["follower\\\\.replication\\\\.throttled\\\\.replicas", "leader\\\\.replication\\\\.throttled\\\\.replicas", "message\\\\.timestamp\\\\.difference\\\\.max\\\\.ms",  "message\\\\.timestamp\\\\.type"]
```

Comment:
a little confusing to me. "to exclude only 'min.insync.replicas' and 'unclean.leader.election.enable" refers to exclude it from the defaults?

Reply:
FYI @staceysalamon-aiven. Yes, that refers to excluding the properties from the defaults.

Reply:
Updated to "For example, to enable replication for..."